README.md

```diff
@@ -1,23 +1,52 @@
+Harassment includes, but is not limited to:
 
-Harassment, hate speech, bullying, sea-lioning, and gas-lighting are prohibited activities and are grounds for suspension from the community when there is clear documentation in public channels. If you experience any of these behaviors in direct messages please immediately contact the administrative team.
+* Verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or preferred technological ecosystem.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Hate speech
+* Sea-lioning ([definition](http://simplikation.com/why-sealioning-is-bad/))
+* Gas-lighting ([definition](https://en.wikipedia.org/wiki/Gaslighting))
+* Spamming of GIFs
+* Advocating for, or encouraging, any of the above behaviour	
 
-Your direct messages are only viewable by yourself and the other involved party and no one else. That being said, this is a community for professional communication and should be treated accordingly, regardless of the number of animated GIFs.
 
-If you have concerns regarding this Slack instance please feel free to contact the administrative team and we will work to understand and address them respectfully and with an appropriate level of privacy to the given situation.
+### Enforcement
+Participants asked to stop any harassing behavior are expected to comply immediately
 
-We hope that you find the community to be enriching to your personal and professional lives, and above all else a place of camaraderie where our collective knowledge and differences are celebrated in a positive, welcoming environment.
+If a participant engages in harassing behaviour, the Administrative Team retain the right to take any actions to keep the event a welcoming environment for all participants. This includes warning the offender or expulsion from the MacAdmins Slack.
 
-Thank you for following this code of conduct. We reserve the right to amend or change the code of conduct at any time and encourage you to periodically review these guidelines to ensure a safe environment for all.
+The Administrative Team may take action to redress anything designed to, or with the clear impact of, disrupting the event or making the environment hostile for any participants.
 
--MacAdmins Slack Administrative Team 
+We expect participants to follow these rules in all public and private channels (including direct messages).
+
+### Reporting
+Reporting violations of these rules should be done directly to a member of the Administrative Team.
```

Comment:
Could say "By contacting them via the #ask-the-slack-admins channel, direct message, or by email listed on their Slack profile and linked at the [Current List of Administrators](https://github.com/macadminsdotorg/slack-assets/blob/master/Admins) page.